docs/docs/documentation/user-guide/create-agents.md

```diff
@@ -62,8 +64,8 @@ Before diving into the implementation, it is important to understand the key con
 
 The LLM (Large Language Model) orchestrating your agent decides which tools to use based on the user's request and the tool descriptions you provide.
 
-:::info
-Solace Agent Mesh provides a set of built-in tools plus built-in support [model context protocol (MCP)](../tutorials/mcp-integration.md) servers which can be configured in the tools list of your agent configuration
+:::tip
+Solace Agent Mesh provides a set of [built-in tools](./builtin-tools/builtin-tools.md) plus out-of-box support for [model context protocol (MCP)](../tutorials/mcp-integration.md) servers which can be configured in the tools list of your agent configuration
```

Comment:
"out-of-the-box", but preferably just delete. It doesn't add anything. Add missing comma and period.

"Solace Agent Mesh provides a set of [built-in tools](./builtin-tools/builtin-tools.md) plus support for [model context protocol (MCP)](../tutorials/mcp-integration.md) servers, which can be configured in the tools list of your agent configuration."